### PR TITLE
Adding README for binaries to OSS

### PR DESCRIPTION
**Pull request by pjh5**

### CODE REVIEW

File: .circleci/README.md [hunk 1]
```diff
@@ -56,3 +56,307 @@ See comment [here](https://github.com/pytorch/pytorch/pull/17323#pullrequestrevi
...
+
+A binary or package (used interchangeably) is a pre-built collection of c++ libraries, header files, python bits, and other files. We build these and distribute them so that users do not need to install from source.
+
+A **binary configuration** is a collection of
+
+* release or nightly
+    * releases are stable, nightlies are beta and built every night
+* python version
+    * linux: 2.7m, 2.7mu, 3.5m, 3.6m 3.7m (mu is wide unicode or something like that. It usually doesn't matter but you should know that it exists)
+    * macos and windows: 2.7, 3.5, 3.6, 3.7
```
Comment (peterjc123) [inline .circleci/README.md]: windows no 2.7

File: .circleci/README.md [hunk 2]
```diff
@@ -56,3 +56,307 @@ See comment [here](https://github.com/pytorch/pytorch/pull/17323#pullrequestrevi
...
+### Where are the binaries?
+
+The binaries are built in CircleCI. There are nightly binaries built every night at 9pm PST (midnight EST) and release binaries corresponding to Pytorch releases, usually every few months.
+
+We have 3 types of binary packages
+
+* pip packages - nightlies are stored on s3 (pip install -f <a s3 url>). releases are stored in a pip repo (pip install torch) (ask Soumith about this)
+* conda packages - nightlies and releases are both stored in a conda repo. Nighty packages have a '_nightly' suffix
+* libtorch packages - these are zips of all the c++ libraries, header files, and sometimes dependencies. These are c++ only
+    * shared with dependencies
```
Comment (peterjc123) [inline .circleci/README.md]: This is the only option for Windows.

**Review by peterjc123 — APPROVED** [review]
Do we need to add the Windows parts here?

**Review by soumith — APPROVED** [review]

**Comment by soumith** [comment]
@peterjc123 once this PR is merged, it'd be good for you to send a PR detailing Windows parts

**Pull request closed without merging**